wiki/faq.md

```diff
@@ -425,13 +426,14 @@ Just try to just wait it out the wallet should automatically fix itself.
 If it still hasn't gone up after a few hours try this:
 
 1.  Close the client
-2.  Clean out your Gridcoin data folder EXCEPT for wallet.dat, your
-    GridcoinResearch.conf file and the folders walletbackups and testnet, if present.
-3.  Restart the wallet and wait for it to sync. If you don't want to wait you can also use the snapshot (File -> Snapshot Download) or you can download [here](https://snapshot.gridcoin.us/snapshot.zip).
+2.  In your Gridcoin data folder, delete the accrual folder, the txleveldb folder, 
+and blk00*.dat (with * being any number).
+3.  Restart the wallet and wait for it to sync. If you don't want to wait you can also use the snapshot
+ which you can use by clicking on File -> Snapshot or using the `-snapshotdownload` [command line argument](cmd-options "wikilink") 
 
 You can also try running the wallet as an administrator.
 
-There is also an [official
+There is also an [
 thread](https://cryptocointalk.com/topic/20303-gridcoin-proof-of-research-connection-sync-problem-thread/)

```

Comment:
this needs to be archived or deleted, as it is no longer accessible

Reply:
removed that